HybridCamLib/src/cam/CamView+Gesture.swift

```diff
@@ -6,32 +6,77 @@ extension CamView: UIGestureRecognizerDelegate {
     * Adds gesture event
     */
    @objc open func addGestureRecognizer() {
-      with(UITapGestureRecognizer(target: self, action: #selector(onPreviewViewTap))) {
-         self.previewView.addGestureRecognizer( $0 )
+      //set up tap gesture recognizers
+      let singleTapGestureRecognizer = {
+         let gesture = UITapGestureRecognizer(target: self, action: #selector(onPreviewViewTap))
+         self.previewView.addGestureRecognizer(gesture)
       }
+      with(UITapGestureRecognizer(target: self, action: #selector(onPreviewViewDoubleTap))) { 
```

Comment:
I just cleaned up this a bit. Hope it compiles 😅

Reply:
Sure, check if double and single tap are still separate from each other because I'm not entirely sure if this line `$0.require(toFail: $0)` works... single taps may react to double taps, too, maybe...

Reply:
Ill make a develop branch. That we can use as a testing ground. Gestures can be tricky.

Reply:
I think it's going to be somewhat hard to keep the single- and double-tap separate from each other due to exactly this line... 🤔 Single-taps need to sort of wait until or rather if another tap is added. If this version doesn't work, I would probably even leave it as I did it since it was working pretty well - just for now. 😅